pkg/network/tracer/tracer_test.go

```diff
@@ -432,22 +432,31 @@ func TestTCPCollectionDisabled(t *testing.T) {
 }
 
 func TestUDPSendAndReceive(t *testing.T) {
-	// incoming.MonotonicSentBytes is 0, when it should be 512
+	t.Run("v4", func(t *testing.T) {
+		testUDPSendAndReceive(t, "127.0.0.1:8001")
+	})
+	t.Run("v6", func(t *testing.T) {
+		testUDPSendAndReceive(t, "[::1]:8001")
```

Comment:
may change the port here (to avoid false positive) as the server will listen on both udp4 and udp6 (UDPServer.network = "udp")